Rakefile

```diff
@@ -486,7 +486,7 @@ task :ci do
       sh 'bundle exec appraisal rails5-mysql2 rake spec:rails'
       sh 'bundle exec appraisal rails5-postgres rake spec:rails'
     end
-  elsif Gem::Version.new('2.4.0') <= Gem::Version.new(RUBY_VERSION)
+  elsif Gem::Version.new('2.4.0') <= Gem::Version.new(RUBY_VERSION) # also covers 2.5
```

Comment:
Even if they're functionally equivalent, I don't think we should merge the 2.4 and 2.5 sections together.

My thoughts here are as we continue to add support for additional future versions of Ruby, each can (and will) support different sets of integrations/tests. Our old CI configuration had this become problem: when trying to minimize duplication by reusing sections for different versions, it made things into a horrible tangle of if/elsif conditions that made the dependencies and task list unintelligible and extremely difficult to update.

To address this, the current configuration is aiming to simplify things by flattening it out by Ruby minor version, at the cost of repetition. Eventually we'll try to reduce this repetition by implementing a configuration file like `travis.yml` works.

@pawelchcki As a proactive measure, can we break these into two sections? 2.4 and 2.5?

Reply:
Sounds good.